examples/citrix_adc/README.md

```diff
@@ -102,7 +86,9 @@ access_token = "<access_token>"
 venafi_api_key = "<venafi_api_key>"
 ```
 
-And finally configure your Citrix infrastructure (these values are illustrative, you should change them accordingly to your own configutation):
+And finally configure your Citrix infrastructure: 
+
+> **NOTE** The values we use here are for illustration only; you should change them according to your own configuration.
```

Comment:
LGTM

Reply:
So I'm more efficient at using BitBucket than I am using GitHub. If I do something I shouldn't, please tell me (<:

So once you review edits here, are you going to accept the PR? I'm thinking you and/or Ricardo should do that. And then also do a PR into master for @tr1ck3r to approve?